### ISSUE TITLE
Error on `connect.mongo.db` when database name has `-` characters 

### ISSUE THREAD

**Issue by manute** [Landoop/stream-reactor#605]
Hi

I have a database name with dashes ,and when the connector it's trying to connect , this error  message is produced: 
```
org.apache.kafka.common.config.ConfigException: connect.mongo.db contains an '-' which are invalid characters for mongo collections
	at com.datamountaineer.streamreactor.connect.mongodb.config.MongoSettings$.apply(MongoSettings.scala:51)
	at com.datamountaineer.streamreactor.connect.mongodb.sink.MongoWriter$.apply(MongoWriter.scala:148)
	at com.datamountaineer.streamreactor.connect.mongodb.sink.MongoSinkTask.start(MongoSinkTask.scala:62)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.initializeAndStart(WorkerSinkTask.java:302)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.execute(WorkerSinkTask.java:191)
	at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:175)
	at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:219)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

Is there a restriction for the `-` char in the database name? 

Because I think `mongodb` has not got this [restriction in the db name](https://docs.mongodb.com/manual/reference/limits/#naming-restrictions) 

Thanks in advance.

**Comment by manute** [comment]
hmmm in fact there's a restriction for dashes in the db name -> https://github.com/Landoop/stream-reactor/blob/master/kafka-connect-mongodb/src/main/scala/com/datamountaineer/streamreactor/connect/mongodb/config/MongoSettings.scala#L50 Why?

**Comment by andrewstevenson** [comment]
@manute This is a bug, it was meant to check the collections but is on the database name by mistake. Do you feel up to submitting a PR?

**Comment by manute** [comment]
sure -> https://github.com/Landoop/stream-reactor/pull/607

**Comment by andrewstevenson** [comment]
Fixed in #607
